### PR TITLE
Deprecation: Changed isChecked and isDisabled to checked and disabled respectively

### PR DESCRIPTION
**Pull request by joschect**

### CODE REVIEW

**Comment by joschect** [comment]
I tried to keep things consistent but I didn't change internal state that used isChecked, only external props. I did change all of the examples to the new prop however.

File: src/components/Check/Check.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ export interface ICheckProps extends React.Props<Check> {
    */
   checked?: boolean;
   /**
-   * isDisabled is deprecated at v.65.1 and will be removed by v 1.0. Use 'checked' instead.
+   * Deprecated at v.65.1 and will be removed by v 1.0. Use 'checked' instead.
```
Comment (dzearing) [inline src/components/Check/Check.tsx]: hmmm where is `@deprecated`

Comment (joschect) [inline src/components/Check/Check.tsx]: Okay I added that!

File: src/components/ContextualMenu/ContextualMenu.Props.ts [hunk 2]
```diff
@@ -110,6 +110,11 @@ export interface IContextualMenuItem {
    * Whether the menu item is disabled
    * @defaultvalue false
    */
+  disabled?: boolean;
+  /**
```
Comment (dzearing) [inline src/components/ContextualMenu/ContextualMenu.Props.ts]: nit add CR

File: src/components/ContextualMenu/ContextualMenu.tsx [hunk 3]
```diff
@@ -294,18 +294,23 @@ export class ContextualMenu extends React.Component<IContextualMenuProps, IConte
...
-        {(hasCheckmarks) ? (
+        { (hasCheckmarks) ? (
           <span
             className={
-              css('ms-ContextualMenu-icon', { 'ms-Icon ms-Icon--CheckMark': item.isChecked, 'not-selected': !item.isChecked })
+              css(
+                'ms-ContextualMenu-icon',
+                {
+                  'ms-Icon ms-Icon--CheckMark': item.isChecked || item.checked,
+                  'not-selected': !item.isChecked && item.checked
```
Comment (dzearing) [inline src/components/ContextualMenu/ContextualMenu.tsx]: normalize the value

File: src/components/DetailsList/DetailsRowCheck.tsx [hunk 4]
```diff
@@ -21,7 +21,7 @@ export const DetailsRowCheck = (props: IDetailsRowCheckProps) => (
     aria-label={ props.ariaLabel }
     >
     { props.canSelect ?
-      <Check isChecked={ props.isSelected } /> :
+      <Check checked={ props.isSelected } /> :
```
Comment (dzearing) [inline src/components/DetailsList/DetailsRowCheck.tsx]: isSelected should be selected

**Comment by joschect** [comment]
@dzearing @ericthompson @micahgodbolt  I think this is all ready, would you guys mind taking another look?

**Pull request merged**